scss/_patterns_pull-quotes.scss

```diff
@@ -1,61 +1,116 @@
+  border: 0;
+  font-size: pow($ms-ratio, 2) * 1rem;
+  margin: $spv-inter--scaleable 0 $spv-inter--scaleable + $sp-unit;
+  overflow: visible;
+  position: relative;
 }
 
 // Pull quote styling
 @mixin vf-p-pull-quotes {
   .p-pull-quote {
-    border: 0;
-    margin: $spv-inter--scaleable 0 $spv-inter--scaleable + $sp-unit;
-    overflow: visible;
     padding: 0 $sph-inter * 2;
-    position: relative;
 
-    > p {
+    &__quote {
+      @extend %vf-heading-4;
+
+      &::before,
+      &::after {
+        font-size: pow($ms-ratio, 2) * 1rem * 2;
+        max-width: pow($ms-ratio, 2) * 1rem;
+        // @media (max-width: $breakpoint-heading-threshold) {
+        //   //font-size: 1.22176rem;
+        // }
+      }
+    }
+
+    &__citation {
       @extend %vf-heading-4;
+    }
+  }
 
-      &:first-of-type::before {
-        @include vf-quotes;
-        content: '\201C\2002'; // Unicode for left double quotation mark +  1/2 em space
-        left: $spv-intra--condensed;
-        top: 0.05rem;
+  .p-pull-quote--small {
+    padding: 0 $sph-inter * 1.5;
 
-        @media (max-width: $breakpoint-heading-threshold) {
-          top: 0.3rem;
-        }
+    .p-pull-quote__quote {
+      font-size: 1rem;
+      line-height: 1.5rem;
+
+      &::before,
+      &::after {
+        font-size: pow($ms-ratio, 2) * 1rem * 1.5;
+        max-width: pow($ms-ratio, 2) * 1rem;
       }
+    }
+
+    .p-pull-quote__citation {
+      font-size: 1rem;
+    }
+  }
+
+  .p-pull-quote--large {
+    padding: 0 $sph-inter * 2.5;
 
-      &:last-of-type {
-        margin-bottom: 0;
+    .p-pull-quote__quote {
+      @extend %vf-heading-3;
+
+      &::before,
+      &::after {
+        font-size: pow($ms-ratio, 2) * 1rem * 2.5;
+        max-width: pow($ms-ratio, 2) * 1rem;
       }
+    }
+
+    .p-pull-quote__citation {
+      @extend %vf-heading-3;
+    }
+  }
+
+  .p-pull-quote,
+  .p-pull-quote--small,
+  .p-pull-quote--large {
+    @include vf-pull-quote;
+  }
 
-      &:last-of-type::after {
-        @include vf-quotes;
-        content: '\2002\201E'; // Unicode for 1/2 em space + low double quotation mark
-        margin-left: $spv-intra--condensed;
-        margin-top: -2.2rem;
+  .p-pull-quote__quote {
+    &:first-of-type::before {
+      @include vf-quote-mark;
+      content: '\201C\2002'; // Unicode for left double quotation mark +  1/2 em space
+      left: $spv-intra--condensed;
+      top: 0.5rem;
 
-        @media (max-width: $breakpoint-heading-threshold) {
-          margin-top: -1.7rem;
-        }
+      @media (max-width: $breakpoint-heading-threshold) {
+        top: 0.75rem;
       }
     }
 
-    &__citation {
-      @extend %vf-heading-4;
-      font-style: italic;
-      margin-top: $spv-intra--condensed--scaleable;
+    &:last-of-type {
+      margin-bottom: 0;
     }
+
+    &:last-of-type::after {
+      @include vf-quote-mark;
+      bottom: 3.75rem;
+      content: '\2002\201E'; // Unicode for 1/2 em space + low double quotation mark
```

Comment:
If you remove the `\2002` here and in the other quote mark I think you won't need to set max-width. It seems weird to have positioning set by the content, margin *and* max-width.